rules/Php81/Rector/ClassMethod/NewInInitializerRector.php

```diff
@@ -191,6 +194,6 @@ private function matchConstructorParams(ClassMethod $classMethod): array
             return [];
         }
 
-        return array_filter($classMethod->params, fn ($v): bool => $v->type instanceof NullableType);
+        return array_filter($classMethod->params, fn ($value): bool => $value->type instanceof NullableType);
```

Comment:
This could be typed to `Param` probably:

```suggestion
        return array_filter($classMethod->params, fn (Param $param): bool => $param->type instanceof NullableType);
```